plugins/net_plugin/net_plugin.cpp

```diff
@@ -612,13 +613,18 @@ namespace eosio {
       }
    };
 
-   struct msgHandler : public fc::visitor<void> {
+   struct msg_handler : public fc::visitor<void> {
       net_plugin_impl &impl;
       connection_ptr c;
-      msgHandler( net_plugin_impl &imp, connection_ptr conn) : impl(imp), c(conn) {}
+      msg_handler( net_plugin_impl &imp, connection_ptr conn) : impl(imp), c(conn) {}
+
+      void operator()( signed_block& msg ) const
```

Comment:
Moving a passed in lvalue reference could break the expectations of the caller. If this was `void operator()( signed_block&& msg ) const` it could be called directly with a moved `signed_block`. Since `fc::static_variant` does not currently support visiting an rvalue reference of a `static_variant`, it does mean the visiting logic at the caller side has to be more complicated.